app/src/main/java/an/maguste/android/navier/api/JsonApiClasses.kt

```diff
@@ -0,0 +1,64 @@
+package an.maguste.android.navier.api
+
+import an.maguste.android.navier.BuildConfig
+import an.maguste.android.navier.data.Genre
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class MoviesDto(
+    val results: List<MovieDto>? = null
```

Comment:
В документации к методу API указано, что всегда возвращается массив объектов
```suggestion
    val results: List<MovieDto>
```